llvm/lib/Target/RISCV/RISCVInstrInfoCOREV.td

```diff
@@ -91,3 +91,85 @@ let Predicates = [HasExtXCoreVHwlp], hasSideEffects = 1, mayLoad = 0, mayStore =
                               "cv.setupi", "$imm1, $imm12, $imm5">,
                 Sched<[]>;
 } // Predicates = [HasExtXCoreVHwlp], hasSideEffects = 1, mayLoad = 0, mayStore = 0
+
+let Predicates = [HasExtXCoreVMac], hasSideEffects = 0, mayLoad = 0, mayStore = 0 in {
+  // 32x32 bit macs
+  def CV_MAC      : RVInstMac<0b0100001, 0b000, (outs GPR:$dest), (ins GPR:$dold, GPR:$src1, GPR:$src2),
```

Comment:
The convention that 'C' set is for the write back register to be called e.g. `rsN_wb`, and the input called `rsN`, so in the case maybe `rD_wb` and `rD`. Also this should have a `let Constraints = '$rsN_wb = $rsN'`  to tie the two registers together. Similarly, the input registers are called `rs1`, `rs2`